api/resources_portal/test/views/test_user.py

```diff
@@ -20,6 +20,9 @@ class TestUserListTestCase(APITestCase):
     def setUp(self):
         self.url = reverse("user-list")
         self.user_data = factory.build(dict, FACTORY_CLASS=UserFactory)
+        # Don't post created_at and updated_at.
```

Comment:
Are we not exposing this just for testing or will we not be able to sort by date attributes on the front end too?

Reply:
This data is for POST. `created_at` and `updated` are read only fields, so they'd be ignored for create or update operations anyway. (see https://www.django-rest-framework.org/api-guide/fields/#read_only)